src/app/rpmostree-builtin-status.c

```diff
@@ -198,29 +196,15 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
       have_any_live_overlay = have_any_live_overlay || have_live_changes;
     }
 
-  if (txn && txn_path)
+  glnx_unref_object RPMOSTreeTransaction *txn_proxy = NULL;
+  if (!rpmostree_transaction_connect_active (sysroot_proxy, NULL, &txn_proxy,
```

Comment:
Hey, Asking a question here if you don't mind :). I noticed in the old commits, the txn_proxy is casted to RPMOSTreeTransactionProxy, while in the new commit, we do not have this cast any more. What might be the benefit behind this?

Reply:
Hm...I think the use of the `Proxy` type before was just a mistake.  You can see we used to cast it to a `Proxy`, then back, without using `Proxy`.

I had to re-read the `gdbus-codegen` stuff a little bit, it looks to me like the `Proxy` types should only be used in special cases, mostly you want to use the interface types.